libs/blocks/action-scroller/action-scroller.css

```diff
@@ -0,0 +1,147 @@
+.action-scroller {
+  --action-scroller-mobile-padding: 50px;
+  --action-scroller-button-color: white;
+  --action-scroller-button-size: 32px;
+  --action-scroller-button-border-color: #646364;
+  --action-scroller-button-border-color-hover: #333;
+  --action-scroller-column-width: calc(var(--action-scroller-item-width) * 1px);
+
+  display: block;
+  position: relative;
+}
+
+.action-scroller .section.scrolling {
+  display: grid;
+  grid-template-columns: repeat(var(--action-scroller-columns), 1fr);
+  grid-auto-columns: minmax(var(--action-scroller-column-width), 1fr);
+  grid-auto-flow: column;
+  padding: 0 var(--action-scroller-mobile-padding);
+  overflow-x: auto;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+  scroll-behavior: smooth;
+
+  --mask-width: 60px;
+  --mask-image-content: linear-gradient(to right,
+      transparent,
+      black var(--mask-width),
+      black calc(100% - var(--mask-width)),
+      transparent);
+  --mask-size-content: 100% 100%;
+  --mask-image-scrollbar: linear-gradient(black, black);
+  --mask-size-scrollbar: 0 100%;
+
+  /* stylelint-disable property-no-vendor-prefix */
+  -webkit-mask-image: var(--mask-image-content), var(--mask-image-scrollbar);
+  -webkit-mask-size: var(--mask-size-content), var(--mask-size-scrollbar);
+  -webkit-mask-position: 0 0, 100% 0;
+  -webkit-mask-repeat: no-repeat, no-repeat;
+  -moz-mask-image: var(--mask-image-content), var(--mask-image-scrollbar);
+  -moz-mask-size: var(--mask-size-content), var(--mask-size-scrollbar);
+  -moz-mask-position: 0 0, 100% 0;
+  -moz-mask-repeat: no-repeat, no-repeat;
+  /* stylelint-enable property-no-vendor-prefix */
```

Comment:
What browsers are this for?  We support the last 2 evergreen versions so I don't think any of this is needed.

Reply:
This is to support the faded edges when an action-item is overflowing in a scrolling area. 
I tested this on a few browsers and wasn't seeing these work w/ out the browser prefixes. 
Correct me if i'm wrong but I thought these are needed for newer css properties like `mask-image`?